src/corefx/System.Globalization.Native/CMakeLists.txt

```diff
@@ -115,5 +115,5 @@ verify_dependencies(
 
 # add the install targets
 install_clr(System.Globalization.Native)
-install(TARGETS System.Globalization.Native_Static DESTINATION .)
+install_clr(System.Globalization.Native_Static)
```

Comment:
This change doesn't seem right. The install_clr strips debug symbols and installs PGO related file. None of this applies to static libraries. Moreover, neither the shared nor the static version of the System.Globalization.Native is needed for the cross components, so I think the whole src/corefx should be excluded for cross components build.